arch/arm/src/stm32/stm32_eth.c

```diff
@@ -634,22 +634,26 @@ struct stm32_ethmac_s
   uint16_t             segments;    /* RX segment count */
   uint16_t             inflight;    /* Number of TX transfers "in_flight" */
   sq_queue_t           freeb;       /* The free buffer list */
-
-  /* Descriptor allocations */
-
-  struct eth_rxdesc_s rxtable[CONFIG_STM32_ETH_NRXDESC];
```

Comment:
Why pull out of the struct and not just add `align` intrude the struct?

Reply:
For consistency with the other STM32's.